Added Google Analytics

13enedict opened the pull request.

Commit:
Added Google Analytics

13enedict committed:
Merge branch 'master' into feature/analytics

Watson6982 commented on .env:
```diff
@@ -1,3 +1,4 @@
-TITLE=techKnow (Dev)
\ No newline at end of file
+TITLE=techKnow (Dev)
+VUE_APP_GOOGLE_ANALYTICS_ID=UA-180455475-1
```
Should this be a variable that can be changed on deployment rather than hardcoded?

13enedict replied:
Thanks, I've made this into a variable

Commit:
Added auto screen tracking

Commit:
Merge branch 'feature/analytics' of https://github.com/lgss/techknow into feature/analytics

ScottyTooHotty committed:
Added banner to home page for cookie consent

Commit:
Made the .env reference to google analytics paramterised

ScottyTooHotty committed:
Merge branch 'feature/analytics' of https://github.com/lgss/techknow into feature/analytics

Commit:
Replaced vue-analytics with vue-gtag

Commit:
Merge branch 'master' into feature/analytics

Commit:
Fixed merge bug

Commit:
Tracking changes

Without explicit consent, there will be GDPR compliant tracking. If consent is given we turn on more explicit tracking. If consent is denied, we turn off all tracking.

Commit:
Fixed setting gtag properties
